test/integration/common

```diff
@@ -227,7 +241,7 @@ download-kubectl() {
 }
 
 start-kind-cluster() {
-    K8SIMAGE=kindest/node:v1.21.1@sha256:fae9a58f17f18f06aeac9772ca8b5ac680ebbed985e266f711d936e91d113bad
+    K8SIMAGE=kindest/node:v1.21.1
```

Comment:
I removed the sha256 pin here because the sha is specific to an architecture, i.e. amd64 and arm64 will have different shas.